tests/test_reports.py

```diff
@@ -29,6 +31,28 @@ def test_get_application_statuses(client):
     )
 
 
+@pytest.mark.parametrize(
+    "query_params",
+    [
+        "?fund_id=47aef2f5-3fcb-4d45-acb5-f0152b5f03c4",
+        "?round_id=c603d114-5364-4474-a0c4-c41cbf4d3bbd",
+        "?fund_id=47aef2f5-3fcb-4d45-acb5-f0152b5f03c4"
+        "&round_id=c603d114-5364-4474-a0c4-c41cbf4d3bbd",
```

Comment:
could we add an application_id to the test?

Reply:
Thanks for the review!  This endpoint in particular doesn't take `application_id`, as it gets the status for all applications with a given `fund_id` or `round_id` - there is however a pre-existing endpoint to get key metrics by application id here: https://github.com/communitiesuk/funding-service-design-application-store/blob/894793c84d1390616569436aa2f310b14116c72c/openapi/api.yml#L287

Reply:
ah ok, thanks for the link. So test coverage for that the search by app functionality is out of scope for this one 👍  Do you think we need to add it to our tech debt?

Reply:
Ah.  I see what you mean now! I can parametrize the existing test for that and make it cover both.

Reply:
thanks for this 👍